tasks/hyrax-metadata-updates/index.js

```diff
@@ -87,14 +88,15 @@ async function getEntryTitle(config, metadata, isUmmG) {
     shortName = metadata.Granule.Collection.ShortName;
     version = metadata.Granule.Collection.VersionId;
   }
-  // Query CMR for collection and retrieve entry title
-  const cmrInstance = new CMR({
-    provider: config.cmr.provider,
-    username: config.cmr.username,
-    passwordSecretName: config.cmr.passwordSecretName,
-    clientId: config.cmr.clientId
+
+  const cmrSettings = await getCmrSettings({
+    ...config.cmr,
+    ...config.launchpad
```

Comment:
What sets `config.launchpad`? I don't see it on the expected task schema

Reply:
Good catch. Didn't realize the hyrax task didn't have it in the config. It is not an optional config param and I've updated the changelog as well.

Reply:
This comment say it is not optional but the changelog/schema still imply that it is optional?

Reply:
I think I meant it is optional, sorry. Definitely optional!